packages/i18n-calypso-cli/package.json

```diff
@@ -0,0 +1,27 @@
+{
+	"name": "i18n-calypso-cli",
+	"version": "4.0.0",
+	"description": "i18n-calypso CLI for extracting translation files from JS sources",
+	"main": "index.js",
+	"repository": {
+		"type": "git",
+		"url": "git+https://github.com/Automattic/wp-calypso.git",
+		"directory": "packages/i18n-calypso-cli"
+	},
+	"author": "@automattic",
+	"license": "GPL-2.0-or-later",
+	"bin": {
+		"i18n-calypso": "cli.js"
+	},
+	"bugs": {
+		"url": "https://github.com/Automattic/wp-calypso/issues"
+	},
+	"homepage": "https://github.com/Automattic/wp-calypso/tree/master/packages/i18n-calypso-cli#readme",
+	"dependencies": {
+		"commander": "^2.9.0",
+		"debug": "^3.2.6",
+		"globby": "^9.0.0",
+		"jed": "1.1.1",
```

Comment:
Any reason to leave `^` (or `~`?) out for `jed`?

Reply:
Thanks for spotting this! `i18n-calypso-cli` actually doesn't need that dependency. Removed in 30923e0 🔪